internal/backend/basesvc/hello.txt

```diff
@@ -1,12 +1,12 @@
 
-        _..--~~~~--.._             autokitteh {{ .Version }}
-     _--              --_          PID:       {{ .PID }}
-    '                    `         gRPC/HTTP: {{ .Addr }}
-   '                      `        {{ .Mode }}
-  '                        `      
+        _..--~~~~--.._             autokitteh   {{ .Version }}
+     _--              --_          PID:         {{ .PID }}
+    '                    `         gRPC/HTTP:   {{ .Addr }}
+   '                      `        UI:          {{ .UIAddr }}
+  '                        `       {{ .Mode }}
  |                    |\   |       
- |                    | \  | 
- |          .,    _,-~   \ | 
+ |                    | \  |       {{ .Temporal1 }}
```

Comment:
Temporal
TemporalUI 

1 and 2 are not informative

Reply:
done